code/game/antagonist/station/revolutionary/excel_faction.dm

```diff
@@ -1,9 +1,22 @@
-/datum/faction/revolutionary
-	id = null
-	faction_invisible = FALSE
-	hud_indicator = "rev"
+/datum/faction/excelsior
+	id = FACTION_EXCELSIOR
+	name = "Excelsior"
+	antag = "infiltrator"
+	antag_plural = "infiltrators"
+	welcome_text = "You are an excelsior infiltrator, here to take over the ship. It will be a long and difficult process.\n\
+	Your strengths lie in manufacturing, production and defense, excelsior are not built for an early rush.\n\n\
+	Your first goal is to retrieve your stash, it contains powerful circuitry you will need to found your base\n\n\
+	After that, your second priority is to establish a secret base, somewhere nobody will detect you. Build your autolathe and teleporter, create turrets and shield generators to defend them. \n\n\
+	Your third goal is to expand. Steal implants, prosthetics or robotic parts and convert them into new implants. recruit new allies and manufacture equipment for them. Send them out to scavenge for useful parts for manufacturing.\n\n\
+	When you are ready, your ultimate goal is to overthrow all the heads of staff, and take control of the ship."
```

Comment:
"You are an Excelsior infiltrator tasked with seizing the CEV Eris. It will be a long and difficult process.\n\
	Your strengths lie in manufacturing, production and defense: Excelsior are not built for an early rush.\n\n\
	Your first goal is to retrieve your stash, as it contains powerful circuitry you will need to start your base.\n\n\
	After that, your second priority is to establish a secret stronghold, where nobody will be able to detect you. Build your autolathe and teleporter, and create turrets and shield generators to defend them. \n\n\
	Your third goal is to expand. Steal implants, prosthetics or robotic parts and convert them into Excelsior implants. Recruit new allies and manufacture equipment for them, then send them out to scavenge for useful parts for fabrication.\n\n\
	When you are ready, your ultimate objective is to overthrow the command structure and take control of the ship."

Reply:
Nikov has given me an alternate for this